backend/src/kittens/tests/kitten-fixture.ts

```diff
@@ -50,9 +59,20 @@ export class KittenFixture {
     expect(this.error?.message).toContain(text);
   }
 
-  thenKittenShouldMatchProperties(properties: Partial<Kitten>): void {
+  thenKittenShouldMatchProperties(properties: Partial<Record<string, any>>): void {
```

Comment:
Code Review: The 'any' type in 'Partial<Record<string, any>>' is too permissive. Consider using a more specific type or a generic type parameter to improve type safety.